src/test/java/Pages/LandingPage.java

```diff
@@ -0,0 +1,121 @@
+    WebElement userBadge;
+    //By userBadge = By.cssSelector("#userBadge.profile");
+    @FindBy (css = "i.fa.fa-plus-circle.create")
+    WebElement playListPlusBtn;
+    //By playListPlusBtn = By.cssSelector("i.fa.fa-plus-circle.create");
+    @FindBy (css = "[data-testid='playlist-context-menu-create-simple']")
+            //failed
+            // li[data-testid='playlist-context-menu-create-simple']
+    WebElement newPlayListBtn;
+    @FindBy (css = "input[name=name]")
+    WebElement newPlayListField;
+    //@FindBy (css = "section#playlists > ul:last-child")
+    //WebElement newPlayListName;
+    @FindBy (css = "section[id='playlists']>ul>li")
+    List<WebElement> playListChildren;
+    @FindBy (css = "section[id='playlists']>ul")
+    WebElement parentOfPlayLists;
+
+    @FindBy (css = ".recent li:nth-child(1) .favorite")
+    WebElement favoriteBtnFirstRecentlyPlayedSongOnHomePage;
+    @FindBy (css = "[href='#!/songs']")
+    WebElement allSongsBtn;
+    @FindBy (css = "tr.song-item:nth-child(1) td.favorite")
+    WebElement firstSongInAllSongs;
+    @FindBy (css = "tr.song-item:nth-child(2) td.favorite")
+    WebElement secondSongInAllSongs;
+    @FindBy (css = "[href='#!/favorites']")
+    WebElement favoritesbtn;
+
+    @FindBy (css = " .virtual-scroller.scroller .item-container table.items tr:nth-child(1)")
+    WebElement firstSongInFavoritesList;
+
+
+
+    /*public WebElement waitForElementToBeClickable(WebElement element) {
+        return wait.until(ExpectedConditions.elementToBeClickable(element));
+    }*/
+
+    public LandingPage clickAddPlaylistPlusButton() {
+        playListPlusBtn.click();
+        return this;
+    }
+
+    public LandingPage nameNewPlayList(String playListName) {
+        newPlayListField.sendKeys(playListName);
+        newPlayListField.sendKeys(Keys.RETURN);
+        return this;
+    }
+    //for By method
+    /*public LandingPage clickAddPlaylistPlusButton() {
+        return wait.until(ExpectedConditions.elementToBeClickable(playListPlusBtn));
+    }*/
+
+    public LandingPage clickAddNewPlaylistButton() {
+        newPlayListBtn.click();
+        return this;
+    }
+
+    public LandingPage getPlayListListSize() {
+        playListChildren.size();
+        return this;
+    }
+
+    public LandingPage clickHeartFirstRecentlyPlayedSongHomePage() {
+        favoriteBtnFirstRecentlyPlayedSongOnHomePage.click();
+        return this;
+    }
+
+    public LandingPage clickAllSongs() {
+        allSongsBtn.click();
+        return this;
+    }
+
+    public LandingPage likeFirstSongInAllSongs() {
+        firstSongInAllSongs.click();
+        return this;
+    }
+
+    public LandingPage likeSecondSongInAllSongs() {
+        secondSongInAllSongs.click();
+        return this;
+    }
+
+    public LandingPage clickFavoritesBtn() {
+        favoritesbtn.click();
+        return this;
+    }
+
+    public boolean firstSongInFavoritesIsDisplayed () {
+        return firstSongInAllSongs;
+    }
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/4e3badf6-df0f-4d22-9111-f88b7ead7f27)

**HINT**

![image](https://github.com/Test-Pro-Bootcamp/QA-Automation-Framework-Template/assets/104761482/3155d0f9-b29b-4ca6-892a-c9a89fe232ec)